README.md

```diff
@@ -1,21 +1,19 @@
-======================
 django CMS Admin Style
 ======================
 
 |pypi| |build| |coverage|
```

Comment:
The buttons break with the file format change.

Reply:
Yeah. I hoped I could just copy and past the tables but now I converted them to rst...